apps/spark/build.gradle

```diff
@@ -14,6 +14,12 @@ configurations {
     shadow.extendsFrom implementation
 }
 
+configurations.all {
+    resolutionStrategy {
+        force 'org.apache.commons:commons-lang3:3.12.0'
+    }
+}
+
```

Comment:
Curious why this is needed

Reply:
Yes this is not required as older version is already excluded and the version 3.12.0 is declared in the convention file.